Source/WebCore/Modules/websockets/WebSocketHandshake.cpp

```diff
@@ -216,17 +216,17 @@ void WebSocketHandshake::reset()
     m_extensionDispatcher.reset();
 }
 
-int WebSocketHandshake::readServerHandshake(const uint8_t* header, size_t len)
+int WebSocketHandshake::readServerHandshake(std::span<const uint8_t> header)
```

Comment:
I didn't change any size here? It was already a size_t. But yes, since the function does return an `int` there is a disk of the size_t not fitting into an int. I'll add an assert.